resources/prometheus/prometheus-rules.yaml

```diff
@@ -176,7 +176,68 @@ spec:
             summary: "Fleetshard synchronizer manages `{{ $value }}` centrals."
             description: "Fleetshard synchronizer manages `{{ $value }}` centrals. The number of Centrals should always be larger than zero in a working system. If it drops to or below zero, fleetshard synchronizer is assumed to be in a failed state."
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-007-fleetshard-sync-reconciliation-error.md"
-
+    - name: rhacs-aws-quota
+      rules:
+        - alert: RHACSCentralDBClustersUtilizationHigh
+          expr: |
+            acs_fleetshard_central_db_clusters_used / acs_fleetshard_central_db_clusters_max >= 0.8
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB clusters is close to its limit."
+            description: "The number of AWS RDS DB clusters is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBInstancesUtilizationHigh
+          expr: |
+            acs_fleetshard_central_db_instances_used / acs_fleetshard_central_db_instances_max >= 0.8
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB instances is close to its limit."
+            description: "The number of AWS RDS DB instances is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBManualSnapshotsUtilizationHigh
+          expr: |
+            acs_fleetshard_central_db_snapshots_used / acs_fleetshard_central_db_snapshots_max >= 0.8
+          for: 5m
```

Comment:
This metric is not influenced by the Probe service creating and deleting instances all the time. The 1h interval is there to prevent the alert firing every time the probe creates an instance (when the current usage is exactly on the threshold).